mibs/adva/8021ae.mib

```diff
@@ -0,0 +1,1951 @@
+-- *****************************************************************
+-- IEEE8021-SECY-MIB
+--
+-- Definitions of managed objects supporting IEEE 802.1AE MACsec.
+--
+-- January 2006
+--
+-- *****************************************************************
+
+IEEE8021-SECY-MIB DEFINITIONS ::= BEGIN
```

Comment:
This mib already exists: ``mibs/IEEE8021-SECY-MIB`.

I expect it's the same for a lot of the others, please remove the duplicates.